modules/log6x-rn-6.0.4.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-4_{context}"]
+= Logging 6.0.4
+
+[id="logging-release-notes-6-0-4-enhancements_{context}"]
+== New features and enhancements
+
+* This enhancement adds `OTel` semantic stream labels to the `lokiStack` output so that you can query logs by using both `ViaQ` and `OTel` stream labels.
+(link:https://issues.redhat.com/browse/LOG-6580[LOG-6580])
+
+[id="logging-release-notes-6-0-4-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the logging `must-gather` did not collect resources such as `UIPlugin`, `ClusterLogForwarder`, `LogFileMetricExporter` and `LokiStack` CR. With this update, these resources are now collected in the respective namespace directory instead of the cluster-logging one.
+(link:https://issues.redhat.com/browse/LOG-6654[LOG-6654])
+
+* Before this update, the operator used a cached client to fetch the `SecurityContextConstraint` cluster resource, which could result in an error when the cache is invalid. With this update, the operator now always retrieves data from the API server instead of using the cache.
+(link:https://issues.redhat.com/browse/LOG-6130[LOG-6130])
+
+* Before this update, the Vector startup script attempted to delete buffer lock files during startup. With this update, the Vector startup script no longer attempts to delete buffer lock files during startup.
+(link:https://issues.redhat.com/browse/LOG-6348[LOG-6348])
+
+* Before this update, a bug in the `must-gather` script for the `cluster-logging-operator` prevented the `LokiStack` from gathering correctly when it existed. With this update, you fix the bug, ensuring that the `LokiStack` gathers correctly.
```

Comment:
I'm not sure, but to me it feels like the first sentence is missing a "being", adjusted the second sentence to match:

```suggestion
* Before this update, a bug in the `must-gather` script for the `cluster-logging-operator` prevented the `LokiStack` from being gathered correctly when it existed. With this update, the `LokiStack` is gathered correctly.
```